src/amp/components/StarRating.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { css, cx } from 'emotion';
-import Star from '@frontend/static/icons/star.svg';
+import { Star } from '@frontend/static/icons/Star';
```

Comment:
I was going to ask to move this into the /components folder but then I realised it's more discoverable here?

Reply:
Yes?